.github/workflows/ci-schedule-imagescanning.yaml

```diff
@@ -0,0 +1,77 @@
+name: image-scanning-on-schedule
+on:
+  schedule:
+    # Run this workflow "At 00:00 UTC on Sunday"
+    - cron: '0 0 * * 0'
+jobs:
+  use-trivy-to-scan-image:
+    name: image scannning
+    # prevent job running from forked repository
+    if: ${{ github.repository == 'karmada-io/karmada' }}
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        # to maintain the latest three release versions, please make timely modifications after the subsequent version is released
+        karmada-version: [ release-1.6, release-1.7, release-1.8 ]
+        include:
+          - karmada-version: release-1.6
+            go-version: 1.20.5
+          - karmada-version: release-1.7
+            go-version: 1.20.6
+          - karmada-version: release-1.8
+            go-version: 1.20.11
+        target:
+          - karmada-controller-manager
+          - karmada-scheduler
+          - karmada-descheduler
+          - karmada-webhook
+          - karmada-agent
+          - karmada-scheduler-estimator
+          - karmada-interpreter-webhook-example
+          - karmada-aggregated-apiserver
+          - karmada-search
+          - karmada-operator
+          - karmada-metrics-adapter
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          # We need to guess version via git tags.
+          fetch-depth: 0

```

Comment:
I guess we don't need to guess the tags, as the image version will be pinned to release branches, like `release-1.6`, right?

Reply:
yes, this parameter can be simply deleted
```suggestion

```